src/test/ui/rfc-2565-param-attrs/param-attrs-cfg.rs

```diff
@@ -0,0 +1,47 @@
+// compile-flags: --cfg something
+// compile-pass
+
+#![deny(unused_variables)]
+#![feature(param_attrs)]
+
+extern "C" {
+    fn ffi(
+        #[cfg(nothing)] a: i32,
+        #[cfg(something)] _b: i32,
```

Comment:
```suggestion
        #[cfg(something)] b: i32, //~ ERROR unused variable: `b`
```
This test would be better if the `#[cfg(something)]` parameters didn't suppress the unused lints with `_`s.